asylo/util/logging.cc

```diff
@@ -134,6 +134,7 @@ bool EnsureDirectory(const char *path) {
 }
 
 bool InitLogging(const char *directory, const char *file_name, int level) {
+  tzset();  // For later use of localtime_r()
```

Comment:
Dropped.

I guess it makes sense that `tzset` might be problematic in a live enclave (I just included it because I found a reference (mentioned in the commit message) that implied that POSIX `localtime` invoked it but `localtime_r` didn't).